src/Observables.jl

```diff
@@ -53,8 +88,9 @@ end
     off(o::AbstractObservable, f)
 
 Removes `f` from listeners of `o`.
+If `raise` == true, throws an error when `f` can't be found.
 """
-function off(o::AbstractObservable, f)
+function off(o::AbstractObservable, f, raise = true)
```

Comment:
Perhaps make into a keyword argument?  Could rename `raise`->`check` perhaps to be a little clearer (same naming as `LinearAlgebra.lu` for a similar purpose; counter example is `Meta.parse(... raise=true)`, but I think `check` is better).